ecs-agent/ecs/model/api/docs-2.json

```diff
@@ -0,0 +1,4197 @@
+        "Resource$integerValue": "<p>When the <code>integerValue</code> type is set, the value of the resource must be an integer.</p>",
+        "Service$desiredCount": "<p>The desired number of instantiations of the task definition to keep running on the service. This value is specified when the service is created with <a>CreateService</a>, and it can be modified with <a>UpdateService</a>.</p>",
+        "Service$runningCount": "<p>The number of tasks in the cluster that are in the <code>RUNNING</code> state.</p>",
+        "Service$pendingCount": "<p>The number of tasks in the cluster that are in the <code>PENDING</code> state.</p>",
+        "ServiceTaskEphemeralStorage$sizeInGiB": null,
+        "ServiceTaskInterfaceConfiguration$index": null,
+        "ServiceTaskInterfaceConfiguration$secondaryPrivateIpAddressCount": null,
+        "TaskDefinition$revision": "<p>The revision of the task in a particular family. The revision is a version number of a task definition in a family. When you register a task definition for the first time, the revision is <code>1</code>. Each time that you register a new revision of a task definition in the same family, the revision value always increases by one. This is even if you deregistered previous revisions in this family.</p>",
+        "TaskSet$computedDesiredCount": "<p>The computed desired count for the task set. This is calculated by multiplying the service's <code>desiredCount</code> by the task set's <code>scale</code> percentage. The result is always rounded up. For example, if the computed desired count is 1.2, it rounds up to 2 tasks.</p>",
+        "TaskSet$pendingCount": "<p>The number of tasks in the task set that are in the <code>PENDING</code> status during a deployment. A task in the <code>PENDING</code> state is preparing to enter the <code>RUNNING</code> state. A task set enters the <code>PENDING</code> status when it launches for the first time or when it's restarted after being in the <code>STOPPED</code> state.</p>",
+        "TaskSet$runningCount": "<p>The number of tasks in the task set that are in the <code>RUNNING</code> status during a deployment. A task in the <code>RUNNING</code> state is running and ready for use.</p>",
+        "Tmpfs$size": "<p>The maximum size (in MiB) of the tmpfs volume.</p>",
+        "Ulimit$softLimit": "<p>The soft limit for the <code>ulimit</code> type.</p>",
+        "Ulimit$hardLimit": "<p>The hard limit for the <code>ulimit</code> type.</p>"
+      }
+    },
+    "InterfaceConfiguration": {
```

Comment:
Same question as [this comment](https://github.com/aws/amazon-ecs-agent/pull/3974#discussion_r1370647495) but for `InterfaceConfiguration`.

Reply:
Same response as [here.](https://github.com/aws/amazon-ecs-agent/pull/3974#discussion_r1370749152)